src/Components/Common/Dropdown.styles.scss

```diff
@@ -4,69 +4,78 @@
   position: relative;
 
   .dropdown-toggle {
-    border: $border;
-    text-transform: uppercase;
+    display: flex;
+    justify-content: space-between;
+    width: 7.5rem;
+
+    font-weight: 500;
+    font-size: $font-size-h5;
+    color: $brand-primary-dark-gray;
 
+    &:hover,
     &.active {
-      border-color: transparent;
+      background: $white-100;
     }
 
-    &::after {
-      display: inline-block;
-      margin-left: .255em;
-      vertical-align: .255em;
-      content: "";
-      border-top: .3em solid;
-      border-right: .3em solid transparent;
-      border-bottom: 0;
-      border-left: .3em solid transparent;
-      box-sizing: border-box;
+    &.active {
+      border-radius: $border-radius-base $border-radius-base 0 0;
+    }
+
+    .icn {
+      height: $size-s;
+      width: $size-s;
+      pointer-events: none;
     }
   }
 
   .list {
```

Comment:
is this har entries list? why do we need an `absolute` position here? if yes,  we could run into situations where adding another component require some unnecessary calculations in the view